### PR TITLE
feat: add leverage module test

### PR DESCRIPTION
**Pull request by RafilxTenfen**
- Added test to check / add registered tokens
> This should be merged after #1 

### How to run 
```shell
$~ make test-leverage-module
```

- If you want to check if the registry actually change, you could not stop the chain (remove the `$(MAKE) stop-chain` from the makefile), run the `test-leverage-module` and query it 
```shell
$~ umeed q leverage registered-tokens --node tcp://0.0.0.0:16657 -o json
```

__Expected Result__

```json
{
  "registry": [
    {
      "base_denom": "uumee",
      "reserve_factor": "0.100000000000000000",
      "collateral_weight": "0.050000000000000000",
      "liquidation_threshold": "0.050000000000000000",
      "base_borrow_rate": "0.020000000000000000",
      "kink_borrow_rate": "0.200000000000000000",
      "max_borrow_rate": "1.500000000000000000",
      "kink_utilization": "0.200000000000000000",
      "liquidation_incentive": "0.100000000000000000",
      "symbol_denom": "UMEE",
      "exponent": 6,
      "enable_msg_supply": true,
      "enable_msg_borrow": true,
      "blacklist": false,
      "max_collateral_share": "0.100000000000000000",
      "max_supply_utilization": "0.500000000000000000",
      "min_collateral_liquidity": "0.750000000000000000"
    }
  ]
}
```

### CODE REVIEW

File: env [hunk 1]
```diff
@@ -1,10 +1,11 @@
-goversion="1.18.1"
-DAEMON=simd
-DENOM=stake
+goversion="1.19"
```
Comment (robert-zaremba) [inline env:1]: We run all things on go 1.18, let's keep 1.18 here until we update other parts.

File: env [hunk 2]
```diff
@@ -1,10 +1,11 @@
...
-DAEMON_HOME=${HOME}/.simd
-GH_URL=https://github.com/cosmos/cosmos-sdk
-CHAIN_VERSION='v0.45.4'
+DAEMON_HOME=${HOME}/.qa-integration/umeed
+GH_URL=https://github.com/umee-network/umee
+CHAIN_VERSION='main'
+IS_PUBLIC=false
 UPGRADE_NAME='v045-to-v046'
 UPGRADE_VERSION='v0.46.0-rc2'
 NUM_TXS=50
```
Comment (robert-zaremba) [inline env:11]: maybe we should create other file: `env-umee` instead of modifying this one?

File: internal/modules/auth/test.py [hunk 3]
```diff
@@ -31,7 +31,7 @@ def test_query_account_fail(self):
         The function `test_query_account_fail` will query wrong account
         and assert fail condition
         """
-        status, _ = query_account("cosmos1xpcfd7pxfmv6gd50y9mwjq50kwqpqrh5mmw72h")
+        status, _ = query_account("umee1xpcfd7pxfmv6gd50y9mwjq50kwqpqrh5mmw72h")
```
Comment (robert-zaremba) [inline internal/modules/auth/test.py:34]: let's move the address to the function argument, this way we will have more flexibility

File: internal/modules/leverage/test.py [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+    def test_query_total_supply(self):
+        status, res = query_registered_tokens()
+        self.assertTrue(status)
+
+        if len(res['registry']) == 0:
+            self.update_leverage_registry()
+
+        status, res = query_registered_tokens()
+        self.assertTrue(status)
```
Comment (robert-zaremba) [inline internal/modules/leverage/test.py:32]: this is already checked in line 26

**Review by robert-zaremba — APPROVED** [review]
utACK

File: env (hunk 2 above)
Comment (RafilxTenfen) [reply env:11]: can be, but then at `env-check.sh` should we source the env-umee ?

File: internal/modules/auth/test.py (hunk 3 above)
Comment (RafilxTenfen) [reply internal/modules/auth/test.py:34]: it is to test the failed query '-' 
The function doesn't have args

File: internal/modules/leverage/test.py (hunk 4 above)
Comment (RafilxTenfen) [reply internal/modules/leverage/test.py:32]: It was called query_registered_tokens twice

File: env (hunk 2 above)
Comment (RafilxTenfen) [reply env]: done at https://github.com/umee-network/qa-integration/pull/2/commits/a76f1dcf2799f30bd2676a118613c9fdd29c2440

**Pull request merged**